docs/elements.md

```diff
@@ -72,6 +72,7 @@ The following **Conditional** elements are available:
 - [`pl-hide-in-panel`](#pl-hide-in-panel-element): Hides content in one or more display panels.
 - [`pl-external-grader-results`](#pl-external-grader-results-element):
   Displays results from questions that are externally graded.
+- [`pl-parsons-problem`](#pl-parsons-problem-element): Submit the result in the correct order and indentation, which by using parson method, 
```

Comment:
Lets put this in the "submission elements" section, since this is something that can be submitted for a grade.